internal/entitlement/repository.go

```diff
@@ -21,3 +23,17 @@ type EntitlementRepo interface {
 	entutils.TxCreator
 	entutils.TxUser[EntitlementRepo]
 }
+
+type CreateEntitlementRepoInputs struct {
+	Namespace       string          `json:"namespace"`
+	FeatureID       string          `json:"featureId"`
+	FeatureKey      string          `json:"featureKey"`
```

Comment:
As above, I don't think this is a good idea, let's have stable references in the DB.

Reply:
see other thread